README.md

```diff
@@ -143,6 +143,16 @@ Make sure you store the yml files shown below in your repository under `.github/
                 ${{ steps.srtool_build.outputs.wasm }}
                 ${{ matrix.chain }}-srtool-digest.json
 
+## Environmental variables
+```
+...
+uses: chevdor/srtool-actions@v*
+env:
+  BUILD_OPTS: --features on-chain-release-build # optional: will be passed to docker srtool run cmd
+with:
+...
+```
```

Comment:
This repo uses ASCIIDOC for the doc and the .md is generated so your change would be overriden.

Could you please add:
```
== Environmental variables

[source,bash]
----
    ...
    uses: chevdor/srtool-actions@v*
    env:
        # optional: will be passed to docker srtool run cmd
        BUILD_OPTS: --features on-chain-release-build
    with:
        ...
----
```

into `README_src.adoc` at L58 ?

You probably don't have the asciidoc toolchain so don't bother installing it, I will generate the new .md after merging.